src/components/SignIn/SignIn.test.tsx

```diff
@@ -21,6 +22,11 @@ const defaults: SignInProps = {
     handleChangeFocusField: jest.fn(),
     changePassword: jest.fn(),
     changeEmail: jest.fn(),
+    captchaType: 'none',
+    renderCaptcha: null,
+    reCaptchaSuccess: false,
+    geetestCaptchaSuccess: false,
```

Comment:
can we combine reCaptchaSuccess and geetestCaptchaSuccess? (we can have only one type of captcha at the same time)